addons/website_profile/controllers/main.py

```diff
@@ -250,16 +252,33 @@ def view_all_users_page(self, page=1, **kwargs):
                 user['karma_gain'] = user_data.get('karma_gain_total', 0)
             user_values.sort(key=itemgetter('position'))
 
+            if (current_user_values):
```

Comment:
if (xxx): -> if xxx:

Plrease reorganize your code if possible.

Can you do this code in the if where you set the current_user_values ?
Why a list if only one element ? 

```python
x = False

if y:
   x = True

zzz

if (x):
   do()
```

Same of:

```python
if y:
    do()
zzz
```